qemu/tests/uefi_check_boot_option.py

```diff
@@ -0,0 +1,59 @@
+import re
+import time
+import logging
+
+from avocado.utils import process
+
+
+from virttest import utils_misc
+from virttest import env_process
+from virttest import error_context
+from tests import uefishell
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verify UEFI config setting in the GUI screen:
+    1) Boot up a guest.
+    2) check splash-time in OVMF.log
+    3) if boot_info not None, check boot entry info
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    def check_log_info(check_info_cmd, expected_result):
+        """
+        info check
+        """
+        output = process.system_output(check_info_cmd).decode()
+        result = re.search(expected_result, output)
+        return result
+
+    check_info_cmd = params["check_info_cmd"]
+    expected_result = params["expected_result"]
+    wait_for_log_timeout = int(params["wait_for_log_timeout"])
+    if params.get("ovmf_log"):
+        error_context.context("Append debugcon parameter to "
+                              "qemu command lines.", logging.info)
+        ovmf_log = utils_misc.get_path(test.debugdir, params["ovmf_log"])
+        params["extra_params"] %= ovmf_log
+        env_process.process(test, params, env,
+                            env_process.preprocess_image,
+                            env_process.preprocess_vm)
+    check_info_cmd %= ovmf_log
+    uefishell_test = uefishell.UEFIShellTest(test, params, env)
```

Comment:
please refer to uefi_boot_from_device.cfg for \"boot from cdrom with_bootindex\"